src/harmony/Switch/Switch.module.css

```diff
@@ -31,7 +34,10 @@
     top: 0;
     height: 100%;
 
-    transition: all 300ms cubic-bezier(0.4, 0, 0.2, 1);
     border-radius: 0.375rem;
     background-color: var(--switch-background-active);
+}
+
+.SwitchPin_animated {
+    transition: all 300ms cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
Use css vars like `--transition-func` and `--transition-time` instead of literal time and function ✌️